File: write-concise-web-tests-with-selenide/src/main/java/de/rieckpil/blog/BookController.java
package de.rieckpil.blog;

import java.util.List;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/api/books")
public class BookController {

  @GetMapping
  public List<Book> getAllBooks() {
    return List.of(new Book(1L, "42", "Effective Java"));
  }
}

File: spring-boot-kotlin-testcontainers/src/main/kotlin/de/rieckpil/blog/BookController.kt
package de.rieckpil.blog

import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
class BookController(private val bookRepository: BookRepository) {

  @GetMapping
  @RequestMapping("/books")
  fun getAllBooks(): List<Book> = bookRepository.findAll()
}
